src/app/models/client-reviews/review.ts

```diff
@@ -0,0 +1,16 @@
+export interface Review {
+    id: number;
+    created_at: string;
+    updated_at: string;
+    review: string;
+    reviewer: Reviewer; 
+    client: number;
+    replies: string[];
+
+}
+export interface Reviewer { 
+    email: string;
```

Comment:
Hey @dorothyas, good work, we should use camelCase for variable naming.

Reply:
@CryceTruly I think the naming here is in snake_case because of how the data comes from the backend, right @dorothyas?

Reply:
@CryceTruly is right. @wangonya, I used to think so as well but it is not dependant on how the data comes from backend.